website/redirects.txt

```diff
@@ -38,3 +38,14 @@
 
 /api.html                 /api/index.html
 /docs/agent/http.html     /api/index.html
+
+/intro/getting-started/install.html      https://learn.hashicorp.com/consul/getting-started/install.html
+/intro/getting-started/agent.html        https://learn.hashicorp.com/consul/getting-started/agent.html
+/intro/getting-started/services.html     https://learn.hashicorp.com/consul/getting-started/services.html
+/intro/getting-started/connect.html      https://learn.hashicorp.com/consul/getting-started/connect.html
+/intro/getting-started/join.html         https://learn.hashicorp.com/consul/getting-started/join.html
+/intro/getting-started/checks.html       https://learn.hashicorp.com/consul/getting-started/checks.html
+/intro/getting-started/kv.html           https://learn.hashicorp.com/consul/getting-started/kv.html
+/intro/getting-started/ui.html           https://learn.hashicorp.com/consul/getting-started/ui.html
+/intro/getting-started/next-steps.html   https://learn.hashicorp.com/consul/getting-started/next-steps.html
+/intro/getting-started/consul.d/web.json https://learn.hashicorp.com/consul/getting-started/consul.d/web.json
```

Comment:
This consul.d and web.json file should just be removed, and we shouldn't maintain redirects for it. Seems like it was just a mistake.

https://github.com/hashicorp/consul/commit/9002bfa37df3aa39024030cb8c312c94cca7c926#diff-a970970e9f0143c37300d2d6ea25d501

Reply:
The redirect to the demo file has been removed as requested. 8a940c69a60